### DISCUSSION TITLE
cmake build of libbaresip only

### DISCUSSION THREAD

**Discussion by juha-h** [baresip/baresip#2207] [General]
How to use cmake to build libbaresip only?

I commenting out these lines in CMakeLists.txt:
```
##############################################################################
# Test and webrtc
#

#add_subdirectory(webrtc)
#add_subdirectory(test)
```
and then those two subdirs were not processed and build worked.

But how to also avoid building baresip binary?  So equivalent of
```
make libbaresip.a
```

**Comment by sreimers** [comment]
You can try:

```cmake -B build -D STATIC=1 && cmake --build build --target baresip -j```

**Comment by juha-h** [comment]
Thanks that was it.  Perhaps also add libbaresip build example to REDME.md?
